Objects/longobject.c

```diff
@@ -4811,7 +4811,7 @@ long_new_impl(PyTypeObject *type, PyObject *x, PyObject *obase)
         return NULL;
     if ((base != 0 && base < 2) || base > 36) {
         PyErr_SetString(PyExc_ValueError,
-                        "int() base must be >= 2 and <= 36");
+                        "int() base must be >= 2 and <= 36  or 0");
```

Comment:
While I proposed exactly this message, when I read again the issue, I prefer Peter's version with a comma. IMHO the comma helps to understand correctly the error :-)
"ValueError: int() base must be >= 2 and <= 36, or 0"
https://bugs.python.org/issue16055#msg171437